src/omdata.h

```diff
@@ -22,6 +22,27 @@ struct overmap_spawns {
     int max_population;
     int chance;
 };
+ //terrain flags enum! this is for tracking the indices of each flag.
+    //is_asphalt, is_building, is_subway, is_sewer, is_ants,
```

Comment:
Oh, that's because I wrote the comment before I wrote the Enum, and never rewrote the comment.